__tests__/app.test.js

```diff
@@ -438,11 +437,54 @@ describe("GET /api/users", () => {
             .get("/api/users")
             .expect(200)
             .then(({ body }) => {
-              console.log(body);
               expect(body.users).toEqual([]);
             });
         });
       });
     });
   });
 });
+
+describe("GET /api/articles?sort_by=created_at&order=DESC", () => {
```

Comment:
I would recommend adding this describe to the describe block of `"GET /api/articles"` as it is under the same endpoint but with added queries